client/client-lib/src/api.rs

```diff
@@ -452,7 +452,7 @@ where
         .await
     }
 
-    async fn fetch_last_epoch(&self) -> FederationResult<u64> {
+    async fn fetch_epoch_count(&self) -> FederationResult<u64> {
         self.request_eventually_consistent("/epoch".to_owned(), erased_no_param())
```

Comment:
We should rename the api call as well while at it.

Reply:
I've now renamed it to `/fetch_epoch_count`. Could also just use `/epoch_count` if you prefer?

Reply:
We'll have to rename all of these soon anyway. There's no reason for them to even start with a `/` to begin with, and I think I'd prefer to make them look as much as bitcoin core as possible, so `getepochcount`. But probably not a job for this PR.

Reply:
OK thanks for explaining that. I'll keep it as `/fetch_epoch_count` for this PR.